app/templates/Membership_Application_Form.html.twig

```diff
@@ -245,6 +245,7 @@
 
 					<div class="container clearfix">
 
+{% if 'PPL' in paymentTypes %}
```

Comment:
Right. I added another commit that allows any combination of payment types and cares for not displaying the list of options if there is only one.